File: jvm/java-blocking-lambda/src/main/java/dev/restate/sdk/examples/LambdaHandler.java
package dev.restate.sdk.examples;

import dev.restate.sdk.lambda.BaseRestateLambdaHandler;
import dev.restate.sdk.lambda.RestateLambdaEndpointBuilder;

public class LambdaHandler extends BaseRestateLambdaHandler {
  @Override
  public void register(RestateLambdaEndpointBuilder builder) {
builder.withService(new Greeter());
  }
}

File: jvm/hello-world-kotlin-lambda/src/main/kotlin/dev/restate/sdk/examples/LambdaHandler.kt
package dev.restate.sdk.examples

import dev.restate.sdk.lambda.BaseRestateLambdaHandler
import dev.restate.sdk.lambda.RestateLambdaEndpointBuilder

class LambdaHandler : BaseRestateLambdaHandler() {
    override fun register(builder: RestateLambdaEndpointBuilder) {
        builder.withService(Greeter())
    }
}
